tests/core/pyspec/eth2spec/test/capella/block_processing/test_process_withdrawals.py

```diff
@@ -75,9 +80,13 @@ def run_withdrawals_processing(spec, state, execution_payload, num_expected_with
     yield 'post', state
 
     if len(expected_withdrawals) == 0:
-        assert state == pre_state
+        next_withdrawal_validator_index = (
+            pre_state.next_withdrawal_validator_index + spec.MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP
+        )
+        assert state.next_withdrawal_validator_index == next_withdrawal_validator_index % len(state.validators)
     elif len(expected_withdrawals) < spec.MAX_WITHDRAWALS_PER_PAYLOAD:
-        assert len(spec.get_expected_withdrawals(state)) == 0
+        bound = min(spec.MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP, spec.MAX_WITHDRAWALS_PER_PAYLOAD)
```

Comment:
this seems like a weird code path. I think we should just consider configs (we make very few) in which the withdrawals per payload is less than the sweep bound as invalid.

Reply:
this is less about the size of the sweep and more just about the scenario where you do the full sweep (of whatever size) and only have a small number of withdrawals

I think all three arms are realistic cases -- do you see it differently?

Reply:
but isn't the bound always going to be `MAX_WITHDRAWALS_PER_PAYLOAD`  here? I mean there is no sense in defining a config with `MAX_VALIDATOR_PER_WITHDRAWAL_SWEEP`  being smaller since then we would never attain the `MAX_WITHDRAWAL_PER_PAYLOAD`  and therefore is equivalent to simply lower it.

Reply:
you are right that for the current parameterization across `mainnet` and `minimal` the bound will always be `MAX_WITHDRAWALS_PER_PAYLOAD` although that is not necessarily the case

we can separately make the commitment that we will never do this but its the type of thing that is hard to track and then enforce...

separately there is the point about this arm of this conditional block and I am imagining a scenario where there are a handful of validators, less than `MAX_WITHDRAWALS_PER_PAYLOAD`, which are withdrawable e.g. going into an inactivity leak